src/lib/core/instance.js

```diff
@@ -17,9 +17,8 @@ class RussInstance {
     try {
       config = require(`${process.cwd()}/.russrc`);
       tasks = fs.readdirSync('russ.tasks');
-    } catch (err) {
-      throw Error('Missing russ files...');
-    }
+    } catch (err) { }
```

Comment:
Not sure `tasks` need to be defined twice here. But, based on #5 this path will likely need to be a variable that's defined based on whether the `config` exists and a key is set or a command line option has been set to define the task directory.